docs/general/networking/nginx.md

```diff
@@ -9,83 +9,145 @@ title: Nginx
 
 ## Nginx from a subdomain (jellyfin.example.org)
 
-:::caution
-
-HTTP is insecure. The following configuration is provided for ease of use only. If you are planning on exposing your server over the Internet you should setup HTTPS. [Let's Encrypt](https://letsencrypt.org/getting-started/) can provide free TLS certificates which can be installed easily via [certbot](https://certbot.eff.org/). Using only HTTP will expose passwords and API keys.
-
-:::
-
 :::tip
 
 The default X-Frame-Options header may cause issues with the webOS app, causing it to remain stuck at a black screen. If enabled, the default Content Security Policy may also cause issues.
 
 :::
 
-Create the file `/etc/nginx/conf.d/jellyfin.conf` which will forward requests to Jellyfin.
+Create the file `/etc/nginx/sites-available/jellyfin` which will forward requests to Jellyfin.  After you've finished, you will need to symlink this file to /etc/nginx/sites-enabled and then reload nginx.  This example assumes you've already acquired certifications as documented in our [Let's Encrypt](https://jellyfin.org/docs/general/networking/letsencrypt#nginx) guide.
+
+Note that a server listening on http port 80 is required for the Certbot / Let's Encrypt certificate renewal process.
+
+### HTTPS config example
 
 ```config
-# Uncomment the commented sections after you have acquired a SSL Certificate
 server {
     listen 80;
     listen [::]:80;
-    # server_name DOMAIN_NAME;
+    server_name jellyfin.DOMAIN.TLD;
```

Comment:
Maybe using `example.com` would be better than `DOMAIN.TLD`? Also add text to tell users to replace the example domain with what they actually want